### ISSUE TITLE
DYK baffles DuckDuckGo

### ISSUE THREAD

**Issue by Anniepoo** [SWI-Prolog/plweb#8]
1. enter site:swi-prolog.org FANN

Expected:
Search results about plfann
Observed:
Every time DYK happened to serve up the FANN entry with some page when ddg was spidering the site.

**Comment by eazar001** [comment]
iirc, this is one of the many flaws that was pointed out with respect to DDG.
A quick workaround search query:
`allintext: swi-prolog.org FANN`.
Of course this won't solve the problem, but you'll get relevant results at least.

**Comment by eazar001** [comment]
Sorry, I actually meant: `inbody: swi-prolog.org FANN`.
The other variant posted above is a google search operator.

**Comment by Anniepoo** [comment]
no, the problem is that FANN appears on pages that aren't related, because it's one of the DYK items.
